spec.html

```diff
@@ -30780,6 +30780,7 @@
         <emu-alg>
           1. Let _obj_ be ? ToObject(*this* value).
           1. Let _len_ be ? ToLength(? Get(_obj_, `"length"`)).
+          1. If _comparefn_ is not *undefined* and IsCallable(_comparefn_) is *false*, throw a *TypeError* exception.
```

Comment:
Since in Firefox, `Array.prototype.sort.call(null)` throws "TypeError: can't convert null to object"  but `Array.prototype.sort.call(null, true)` throws "TypeError: invalid Array.prototype.sort argument", this matches web reality. Changing now.